src/unitxt/operators.py

```diff
@@ -758,7 +758,7 @@ def process(
 
 
 class CopyFields(FieldOperator):
-    """Copies specified fields from one field to another.
+    """Copies values from specified fields to specified fields.
```

Comment:
Hi. I think it will be helpful to add examples, as this is a common operator.  Include the use of advanced options like use_query=True.